setup.sh

```diff
@@ -101,6 +58,27 @@ while test $# -gt 0; do
     shift
 done
 
+# Constants
+PATH="${HOME}/.wks/bin:${PATH}"
```

Comment:
making this constant instead of conditional on the `--no-download` flag is a notable change in this patch.